docs/esterni/piano_di_qualifica/piano_di_qualifica.tex

```diff
@@ -30,6 +30,7 @@
 \addversione{2.4.0}{20/04/2022}{Alberto Nicoletti}{Redazione}{Aggiornamento per ciclo 4.}
 \addversione{2.4.1}{21/04/2022}{Elia Scandaletti}{Redazione}{Correzione grammatica.}
 \addversione{2.5.0}{01/05/2022}{Raffaele Oliviero}{Redazione}{Aggiornamento per ciclo 5.}
+\addversione{2.6.0}{04/05/2022}{Andrea Volpe}{Redazione}{Aggiornamento per ciclo 6.}
```

Comment:
```suggestion
\addversione[Raffaele Oliviero]{2.6.0}{04/05/2022}{Andrea Volpe}{Redazione}{Aggiornamento per ciclo 6.}
```
Con il nuovo sistema di versionamento si possono aggiungere i revisori in questo modo. Ovviamente il file non compilerà finchè non verrà effettuato il merge sul main del nuovo sistema. Se non sbaglio nelle norme di progetto il file deve compilare perchè venga effettuato un merge, quindi siamo in un po' in un edge case, vedi te se fare questa modifica

Reply:
Probabilmente è più consono alle norme mettere la versione così com'è adessso e in un secondo momento aggiornare i file di cui si sa il revisore